src/QMCDrivers/QMCDriverNew.cpp

```diff
@@ -99,9 +99,10 @@ QMCDriverNew::QMCDriverNew(const ProjectData& project_data,
 // To work around the issue, check the local pointer for nullptr before restoring to global storage.
 QMCDriverNew::~QMCDriverNew()
 {
+  auto& rng_children = RandomNumberControl::getChildren();
```

Comment:
This is still awful because we're tying QMCDriverNew to a static object. I don't think it should know RandomNumberControl at all. But this PR doesn't bake that in any further.

Reply:
I'm actually working on this. Moving the ownership of RNGs back and forth is really bad and I'm going to make the driver side keeping reference without changing ownership.